aqueduct_client/examples/Sample Aqueduct Usage.ipynb

```diff
@@ -0,0 +1,533 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style>\n",
+       "    .dataframe thead tr:only-child th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: left;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>created_at</th>\n",
+       "      <th>id</th>\n",
+       "      <th>name</th>\n",
+       "      <th>asset_identifier_format</th>\n",
+       "      <th>start_date</th>\n",
+       "      <th>end_date</th>\n",
+       "      <th>status</th>\n",
+       "      <th>output_format</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>760</th>\n",
+       "      <td>2020-01-08T18:28:51.652528</td>\n",
+       "      <td>5e161f63a809d1660ad190ea</td>\n",
+       "      <td>MyPipeline</td>\n",
+       "      <td>sid</td>\n",
+       "      <td>2015-12-31</td>\n",
+       "      <td>2017-12-31</td>\n",
+       "      <td>SUCCESS</td>\n",
+       "      <td>csv</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                     created_at                        id        name  \\\n",
+       "760  2020-01-08T18:28:51.652528  5e161f63a809d1660ad190ea  MyPipeline   \n",
+       "\n",
+       "    asset_identifier_format  start_date    end_date   status output_format  \n",
+       "760                     sid  2015-12-31  2017-12-31  SUCCESS           csv  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "all_pipelines = client.get_all_pipeline_executions()\n",
+    "\n",
+    "historical_pipelines = pd.DataFrame(all_pipelines).sort_values(\n",
+    "    \"created_at\", ascending=False\n",
+    ")\n",
+    "historical_pipelines = historical_pipelines[\n",
+    "    [\n",
+    "        \"created_at\",\n",
+    "        \"id\",\n",
+    "        \"name\",\n",
+    "        \"asset_identifier_format\",\n",
+    "        \"start_date\",\n",
+    "        \"end_date\",\n",
+    "        \"status\",\n",
+    "        \"output_format\",\n",
+    "    ]\n",
+    "]\n",
+    "\n",
+    "historical_pipelines.head(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Historical Aqueduct outputs can be retrieved by selecting the **id's** shown above. Below is an example:"
```

Comment:
--> "Historical, point-in-time Aqueduct results can be retrieved by selecting the **IDs** shown above. Below is an example:"